File: Comtrade.java
package ru.gur;

public class Comtrade {

    private StringBuilder dat = new StringBuilder();

    private String cfg;

    public Comtrade() {
    }

    public Comtrade(String cfg, StringBuilder dat) {
        this.cfg = cfg;
        this.dat = dat;
    }

    public StringBuilder getDat() {
        return dat;
    }

    public String getCfg() {
        return cfg;
    }

    public void setCfg(String cfg) {
        this.cfg = cfg;
    }

    public void setDat(StringBuilder dat) {
        this.dat = dat;
    }
}

File: python/Comtrade.py
class Comtrade:

    def __init__(self, cfg='', dat=''):
        self._cfg = cfg
        self._dat = dat

    def get_cfg(self):
        return self._cfg

    def get_dat(self):
        return self._dat

    def set_cfg(self, x):
        self._cfg = x

    def set_dat(self, x):
        self._dat = x

